### ISSUE TITLE
Shortlink automation

### ISSUE THREAD

**Issue by EthanThatOneKid** [acmcsufoss/acmcsuf.com#213]
### What's your idea?

Similar to #54, we can create a workflow that creates a new pull request updating the shortlink data used by the website. Currently, if a user lands on https://acmcsuf.com/ with a pathname matching any of the names of the predefined shortlinks (in `/lib/constants/links.ts`), it is redirected to the assigned destination. For example, `https://acmcsuf.com/discord` redirects to the Discord server’s invite link.

This means each time we want to add a new shortlink or remove an old one from the website, we must manually create a PR for this change. However, this can be easily automated via issue form.

**Comment by EthanThatOneKid** [comment]
Closing/reopening this issue to populate tools. More info: #776.

**Comment by k4rni** [comment]
Status: blocked. Waiting on a stable version of [codemod](https://oss.acmcsuf.com/codemod).

**Comment by k4rni** [comment]
Resolved: https://oss.acmcsuf.com/shorter 

More information: https://acmcsuf.com/shorter-handbook
